test/jdk/java/net/vthread/BlockingSocketOps.java

```diff
@@ -668,191 +712,33 @@ public void close() throws IOException {
         }
     }
 
-    /**
-     * Closes a socket after a delay
-     */
-    static class ScheduledCloser implements Runnable {
-        private final Closeable c;
-        private final long delay;
-        ScheduledCloser(Closeable c, long delay) {
-            this.c = c;
-            this.delay = delay;
-        }
-        @Override
-        public void run() {
-            try {
-                Thread.sleep(delay);
-                c.close();
-            } catch (Exception e) { }
-        }
-        static void schedule(Closeable c, long delay) {
-            new Thread(new ScheduledCloser(c, delay)).start();
-        }
+    @FunctionalInterface
+    interface ThrowingRunnable {
+        void run() throws Exception;
     }
 
     /**
-     * Interrupts a thread after a delay
+     * Runs the given task asynchronously after the current virtual thread has
+     * parked, or after a max wait time, whichever is first.
```

Comment:
Does this comment need an update to match the current implementation? From what I can see in the current implementation of `runAfterParkedAsync`, in this PR, it doesn't have a "max wait time" construct before triggering the task.

Reply:
Well spotted, that comment is out of date now.